src/tests/test_utils/common_test_utils/include/common_test_utils/matcher.hpp

```diff
@@ -3,6 +3,8 @@
 //
 #pragma once
 
+#pragma once
+
 #include "ngraph/log.hpp"
```

Comment:
and no namespace... Please add

Reply:
Probably this is due to rebase side effects. After last rebase second `#pragma once` gone